src/main/java/ch/uzh/ifi/hase/soprafs22/rest/dto/UserGetPreferencesDTO.java

```diff
@@ -0,0 +1,38 @@
+package ch.uzh.ifi.hase.soprafs22.rest.dto;
+
+
+import ch.uzh.ifi.hase.soprafs22.constant.Gender;
+
+import java.util.List;
+
+public class UserGetPreferencesDTO {
+
+    private int minAge;
+    private int maxAge;
+    private List<Gender> genderPreferences;
+
+
+    public int getMinAge() {
+        return minAge;
+    }
+
+    public void setMinAge(int minAge) {
+        this.minAge = minAge;
+    }
+
+    public int getMaxAge() {
+        return maxAge;
+    }
+
+    public void setMaxAge(int maxAge) {
+        this.maxAge = maxAge;
+    }
+
+    public List<Gender> getGenderPreferences() {
+        return genderPreferences;
+    }
+
+    public void setGenderPreferences(List<Gender> genderPreferences) {
+        this.genderPreferences = genderPreferences;
+    }
```

Comment:
Does this work? Have you tested it in Postman? (I remember you having troubles with Gender in User)

Reply:
needs adjusting. Will do it on the weekend

Reply:
Actually seems to be working but will check again on sunday if i have time. Dont merge yet

Reply:
Works without a problem now. Can be passed as array of strings from Client.

Reply:
Nice!